### PR TITLE
Update copyright year to 2022

### PR DESCRIPTION
**Pull request by doveye**
Closes: #882
Signed-off-by: Esther <doveye@uk.ibm.com>

### CODE REVIEW

**Comment by doveye** [comment]
Jenkins doc stage

**Comment by doveye** [comment]
Build ran without error, docs look fine at https://eclipse-openj9.github.io/openj9-docs-staging/885/openj9_support/ and I can see the updated copyright in "View page source".

@keithc-ca could you please merge this PR, thanks.

File: docs/attachapi.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/attachapi.md:2]: The first year should be 2020 - this file was first added in that year.

File: docs/jitserver_tuning.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/jitserver_tuning.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/version0.29.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/version0.29.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/version0.30.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2022 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/version0.30.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/version0.29.1.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/version0.29.1.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxdiagnosesynconvaluebasedclasses.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxdiagnosesynconvaluebasedclasses.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxenable3164interoperability.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ﻿<!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxenable3164interoperability.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxensurehashed.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxensurehashed.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxjitserveraddress.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxjitserveraddress.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxjitserverlocalsynccompiles.md [hunk 10]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxjitserverlocalsynccompiles.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxjitserverlogconnections.md [hunk 11]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxjitserverlogconnections.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxjitserverport.md [hunk 12]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxjitserverport.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxjitservershareromclasses.md [hunk 13]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxjitservershareromclasses.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxjitserversslcert.md [hunk 14]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxjitserversslcert.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxjitservertimeout.md [hunk 15]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxjitservertimeout.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxrequirejitserver.md [hunk 16]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxrequirejitserver.md:2]: The first year should be 2021 - this file was added in that year.

File: docs/xxusejitserver.md [hunk 17]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2021, 2021 IBM Corp. and others
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/xxusejitserver.md:2]: The first year should be 2021 - this file was added in that year.

**Review by keithc-ca — CHANGES_REQUESTED** [review]
Please don't modify the first copyright year unless it's clear that it was wrong.

**Comment by keithc-ca** [comment]
As discussed in #882, I don't agree with the guidelines, but until we have agreement to change those guidelines, we'll have to follow them.

File: buildenv/Dockerfile [hunk 18]
```diff
@@ -1,4 +1,4 @@
-# (C) Copyright IBM Corporation 2016, 2021
+# (C) Copyright IBM Corporation 2016, 2022
```
Comment (keithc-ca) [inline buildenv/Dockerfile:1]: Please update the copyright notice with the text used everywhere else.

File: docs/builds.md [hunk 19]
```diff
@@ -1,4 +1,26 @@
-
+<!--
+* Copyright (c) 2021, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/builds.md:2]: This should say '2017, 2022' if we're following the guidelines.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: buildenv/Dockerfile (hunk 18 above)
Comment (doveye) [reply buildenv/Dockerfile]: Done, good spot

File: docs/builds.md (hunk 19 above)
Comment (doveye) [reply docs/builds.md]: Yep, done, thanks

File: buildenv/Dockerfile (hunk 18 above)
Comment (keithc-ca) [reply buildenv/Dockerfile]: I see the dates have changed, but I still see non-standard text.

Comment (doveye) [reply buildenv/Dockerfile]: Yes, I only looked at the years. Taking a closer look, the rest of the text is quite different from the text in other files, which is odd. I see from the history that it was the same originally but changed when a new Dockerfile was created to eventually replace the original. The new legal text was probably an inadvertent change (it's not mentioned in the update). On that basis, I've changed it back to match the other files.

Comment (keithc-ca) [reply buildenv/Dockerfile]: This file must continue to use the comment style understood by docker (lines beginning with `#`).

Comment (doveye) [reply buildenv/Dockerfile]: Argh. Glad you spotted that because I wouldn't have (clearly)! Now fixed.

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
jenkins doc stage

**Pull request merged**